data/ZendeAditya.json

```diff
@@ -1,7 +1,15 @@
 {
   "name": "Aditya Zende",
   "type": "personal",
-  "bio": "A freelance web developer",
+  "bio": "**A freelance web developer** Technical Writer",
+  "tags":[
+  "open source",
+  "Web developer",
+  "Technical writer",
+  "javascript",
+  "React",
+  "Tailwind"
+  ]
```

Comment:
```suggestion
  ],
```

Reply:
Yeah that's the only issue and everything would be fine